### PR TITLE
fix rmsprop learning rate for convergence

### PR DESCRIPTION
**Pull request by murasaki0110**
Rmsprop with default learning rate (0.001) cannot converge in this example. 
Initialize learning rate to (0.0001) and add weight decay fix the problem.

https://github.com/fchollet/keras/issues/5239
As seen in this issue also, the loss will decrease at the begining but start increasing after few epochs due to too large learning rate.
Solution to this is decreasing the learning rate with rmsprop as I supposed or changing the optimizer like Adam or SGD.

### CODE REVIEW

**Comment by dlcheng** [comment]
Same problem here, change to adam works fine.
